src/cljs/main/broadfcui/page/workspace/monitor/common.cljs

```diff
@@ -27,7 +27,9 @@
 (def call-running-statuses #{"NotStarted" "Starting" "Running"})
 (def call-failure-statuses #{"Failed" "Aborted"})
 
-(def google-cloud-context "https://console.cloud.google.com/storage/browser/")
+(defn google-account-chooser-prefix [destination] (str "https://accounts.google.com/AccountChooser?continue=" destination))
```

Comment:
this wasn't originally in the scope of the ticket but i cleared it w/ PO.

we've gotten too many questions over the years about why google says a user doesn't have permission to view these links only to find out that google defaulted to the wrong account. forcing the account chooser adds an extra step to the flow but prevents these types of questions from eating up debugging time.